ph_rando/patcher/_util.py

```diff
@@ -397,6 +398,69 @@ def _patch_bmg_events(areas: list[Area], input_rom: rom.NintendoDSRom) -> None:
             )
 
 
+def _patch_system_bmg(input_rom: rom.NintendoDSRom) -> None:
+    """
+    Patches the system BMG file to include all non-global "got item" BMG messages,
+    effectively making them global.
+    The "got item" message is the text that pops up when Link collects an item.
+    """
+    gims = list()
```

Comment:
Minor thing, but the convention elsewhere in the codebase is to use `[]` for instantiating an empty list.
```suggestion
    gims = []
```